micromasters/models.py

```diff
@@ -27,7 +28,7 @@ def update(self, **kwargs):
         database level without loading objects into memory.
         """
         if "updated_on" not in kwargs:
-            kwargs["updated_on"] = datetime.datetime.utcnow()
+            kwargs["updated_on"] = datetime.datetime.now(tz=pytz.UTC)
```

Comment:
In [settings.py](https://github.com/mitodl/micromasters/blob/master/micromasters/settings.py#L290)